www/universal_links.js

```diff
@@ -79,14 +79,14 @@ var universalLinks = {
    *
    * @param {number} milliseconds - Optional. The number of milliseconds to wait before executing the code. If omitted, the value 0 is used
    */
-  checkDeepLink: function (milliseconds = 2000) {
+  checkDeepLink: function () {
       var _this = this;
       return new Promise(function (resolve, reject) {
           setTimeout(function () {
               if (_this.dpLink)
                 _this.validateDeeplink()
               resolve(_this.dpLink);
-          }, milliseconds);
+          }, milliseconds || 0);
```

Comment:
Why not just change milliseconds default value to 0?

Reply:
Because function default value also a feature introduced in ES6/ES2015 and thus not supported in older browsers like the webview on Android 5.1.

First changed the arrow functions and then it died on this.